### PR TITLE
Make `guest` a reserved word that can't be used for OpenVPN credentials.

### PR DESCRIPTION
**Pull request by Yiin**
Fixes https://gitlab.protontech.ch/ProtonVPN/web/protonvpn-settings/issues/23

### CODE REVIEW

File: containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ const OpenVPNCredentialsModal = ({ username = '', password = '', fetchUserVPN, .
     const handleChangePassword = ({ target }) => setCredentials({ ...credentials, password: target.value });
 
     const handleSubmit = async () => {
+        const RESERVED_WORDS = ['guest'];
+        if (RESERVED_WORDS.includes(credentials.username)) {
```
[the remark this answers is not in the document]
Comment (Yiin) [reply containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js]: Good point, it probably does

File: containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js [hunk 2]
```diff
@@ -25,6 +25,13 @@ const OpenVPNCredentialsModal = ({ username = '', password = '', fetchUserVPN, .
     const handleChangePassword = ({ target }) => setCredentials({ ...credentials, password: target.value });
 
     const handleSubmit = async () => {
+        const RESERVED_WORDS = ['guest'];
```
Comment (EpokK) [inline containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js]: What about moving `RESERVED_WORDS` outside of the `handleSubmit` function?

Comment (Yiin) [reply containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js]: I thought about moving it outside but wanted to keep it close to the context it's used in (as it's not being reused anywhere else). RESERVED_USERNAMES definitely sounds better :)

**Comment by emnproton** [comment]
Verified as fixed.

**Pull request merged**